docs/site/src/pages/style/color/Color.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import * as colors from 'material-ui/styles/colors';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import { getContrastRatio } from 'material-ui/styles/colorManipulator';
+
+const mainColors = [
+  'Red', 'Pink', 'Purple', 'Deep Purple', 'Indigo', 'Blue', 'Light Blue',
+  'Cyan', 'Teal', 'Green', 'Light Green', 'Lime', 'Yellow', 'Amber', 'Orange',
+  'Deep Orange',
+];
+const neutralColors = ['Brown', 'Grey'];
+
+export const styleSheet = createStyleSheet('colors', (theme) => ({
+  name: {
+    display: 'block',
+    marginBottom: 60,
+  },
+  colorContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  blockSpace: {
+    height: '4px',
+    color: '#eeeeee',
+  },
+  colorValue: {
+    fontSize: 12,
+  },
+  root: {
+    marginLeft: '50px',
+    boxSizing: 'initial',
+  },
+  hex: {
+    float: 'right',
+  },
+  colorGroup: {
+    float: 'left',
```

Comment:
We should no longer need any float layout model. Adding on the parent sounds better:
```js
    display: 'flex',
    flexWrap: 'wrap',
```